Fire Emblem/Fire-emblem.html

```diff
@@ -800,89 +760,71 @@
-                                   <input type="number" name="attr_ride_points" style="width:145px; height: 25px; margin: 0px 0px 0px 5px; padding:1px 3px 0px; font-size: 14px; text-align: center;" value="0" />    
+                                   <input type="number" name="attr_ride_total" style="width:145px; height: 25px; margin: 0px 0px 0px 5px; padding:1px 3px 0px; font-size: 14px; text-align: center;"/>
+                                   <input type="text" name="attr_ride_stats" style="width:225px; height: 25px; margin: 0px 5px 0px 5px; padding:1px 3px 0px; font-size: 14px; text-align: center;" value="STR or SPD + LUK/2" readonly/> 
+                                   <input type="number" name="attr_ride_level" style="width:145px; height: 25px; margin: 0px 0px 0px 5px; padding:1px 3px 0px; font-size: 14px; text-align: center;"/>    
                               </div><!-- end .sheet-ampladatotal30 --> 
                     </div><!-- end .sheet-width100 -->
                      
                     <div class="sheet-width100">    
                               <div class="sheet-ampladatotal30">
                                    <input type="checkbox" class="sheet-mycheck" >
                                    <input type="text" name="attr_sleight" style="width:220px; height: 25px; margin: 0px 0px 0px 5px; padding:1px 3px 0px; font-size: 14px; text-align: center;" value="SLEIGHT OF HAND" readonly />
-                                   <button title="Sleight of hand roll" class="default" type= "roll" name= "roll_sleight" value= "&{template:default} {{name=ROLL SLEIGHT (@{character_name})}} {{1d100 + Total =[[@{sleight_total}]] }} {{ [[1d100+@{sleight_total}]] }}"></button>
-                                   <input type="number" name="attr_sleight_total" style="width:145px; height: 25px; margin: 0px 0px 0px 5px; padding:1px 3px 0px; font-size: 14px; text-align: center;" value="@{sleight_statsnumber}+@{sleight_points}" disabled="true" />
+                                   <input type="number" name="attr_sleight_total" style="width:145px; height: 25px; margin: 0px 0px 0px 5px; padding:1px 3px 0px; font-size: 14px; text-align: center;"/>
                                    <input type="text" name="attr_sleight_stats" style="width:225px; height: 25px; margin: 0px 5px 0px 5px; padding:1px 3px 0px; font-size: 14px; text-align: center;" value="SKL + LUK/2" readonly/> 
-                                  <input type="hidden" name="attr_sleight_statsnumber" style="width:95px; height: 25px; margin: 0px 0px 0px 5px; padding:1px 3px 0px; font-size: 14px; text-align: center;" value="[[@{skl}+@{luk}/2]]" disabled="true" />
-                                   <input type="number" name="attr_sleight_points" style="width:145px; height: 25px; margin: 0px 0px 0px 5px; padding:1px 3px 0px; font-size: 14px; text-align: center;" value="0" />    
```

Comment:
A lot of the `_statsnumber` and `_points` attrs are being removed, and it appears they're being replaced with `_level`? I'm not sure what their function is, but we can't approve an update that would delete players' data without handlers to convert it into their replacements.

If these are somehow just UI attributes, I'll be happy to approve this after an explanation, but the names lead me to believe they have some mechanical importance.